src/include/storage/index/bplustree.h

```diff
@@ -11,22 +11,12 @@
 #include <vector>
 
 #include "common/shared_latch.h"
+#include "loggers/index_logger.h"
 #include "storage/index/index.h"
 #include "storage/index/index_defs.h"
 
-#include "loggers/index_logger.h"
-
 namespace noisepage::storage::index {
 
-// If node size goes above this then we split it
-// Thresholds for Inner Node
-#define INNER_NODE_SIZE_UPPER_THRESHOLD ((int)128)
```

Comment:
These should have been `constexpr` but I guess we missed that on review. We'll do a `#define` sweep of the codebase again at a later date.